uflash.py

```diff
@@ -242,13 +242,16 @@ def save_hex(hex_file, path):
         output.write(hex_file.encode('ascii'))
 
 
-def flash(path_to_python=None, paths_to_microbits=None, path_to_runtime=None):
+def flash(path_to_python=None, python_script=None,
```

Comment:
Adding `python_script` in the middle of the function signature would break any application importing uFlash and using ordered arguments, probably better to keep it at the end for backwards compatibility.